cinema/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+
+class Movie(models.Model):
+
+    title = models.CharField(max_length=200)
+    description = models.TextField()
+    duration = models.IntegerField()
```

Comment:
Consider using `PositiveIntegerField` instead of `IntegerField` for the `duration` field to ensure that only positive values are allowed, as duration should not be negative.